packages/fluxcd-source-controller/0.36.1-build.2/Dockerfile

```diff
@@ -0,0 +1,126 @@
+# ARG BASE_VARIANT=alpine
+ARG BASE_VARIANT=bullseye
+ARG GO_VERSION=1.17
+ARG XX_VERSION=1.1.0
+
+ARG LIBGIT2_IMG=ghcr.io/fluxcd/golang-with-libgit2
```

Comment:
Please review Flux `v0.36.x` release notes. `libgit2` is no longer supported in this version of Flux SC. Please remove this.

Reply:
@rashedkvm this is not related to Flux SC but is related to the build process of caymanization. And this Dockerfile is not being referenced anywhere internally.

Reply:
@rashedkvm use of `libgit2` in this Dockerfile is completely unrelated to Flux SC deprecating the support for it.

Reply:
@hemakshis can it be removed if it is not used?

Reply:
Please review how `libgit2` is used in this build, especially at this line 
```
COPY --from=libgit2-libs /usr/local/ /usr/local/
```

Reply:
After chatting with @hemakshis - the `libgit2` lib is not used for Source Controller runtime. It is used for the internal build systems. In that case, this can be remain as-is